src/org/labkey/test/pages/EditDatasetDefinitionPage.java

```diff
@@ -15,267 +15,258 @@
+//        WebElement cohortAssociation = Locator.xpath("//tr/td[@class='labkey-form-label'][descendant::div[text()='Cohort Association' and @class='gwt-Label']]/following-sibling::td/descendant::em").findWhenNeeded(this);
+//        WebElement additionalKeyNoneRadio = Locator.radioButtonById("button_none").findWhenNeeded(this);
+//        WebElement additionalKeyDataFieldRadio = Locator.radioButtonById("button-datafield").findWhenNeeded(this);
+//        WebElement additionalKeyManagedFieldRadio = Locator.radioButtonById("button_managedField").findWhenNeeded(this);
+//        WebElement additionalKeyDataFieldSelect = Locator.name("list_dataField").findWhenNeeded(this);
+//        WebElement additionalKeyManagedFieldSelect = Locator.name("list_managedField").findWhenNeeded(this);
+//        WebElement demographicsCheckbox = Locator.checkboxByName("demographicData").findWhenNeeded(this);
+//        WebElement demographicsSharedBy = Locator.name("demographicsSharedBy").findWhenNeeded(this);
+//        WebElement showInOverviewCheckbox = Locator.checkboxByName("showByDefault").findWhenNeeded(this);
+//        WebElement importFieldsButton = Locator.lkButton("Import Fields").findWhenNeeded(this);
+//        WebElement exportFieldsButton = Locator.lkButton("Export Fields").findWhenNeeded(this);
+//        WebElement inferFieldsButton = Locator.lkButton("Infer Fields from File").findWhenNeeded(this);
+//
+//        WebElement applyChosenFieldTypeButton = Locator.button("Apply").findWhenNeeded(this);
+//        WebElement cancelChosenFieldTypeButton = Locator.button("Cancel").findWhenNeeded(this);
+//
+//        WebElement lookupFieldTypeFolder = Locator.name("lookupContainer").findWhenNeeded(this);
+//        WebElement lookupFieldSchema = Locator.name("schema").findWhenNeeded(this);
+//        WebElement lookupFieldTable = Locator.name("table").findWhenNeeded(this);
+//
+//        WebElement additionalKeyManagedKeyParentSpan = Locator.xpath("//span[descendant::input[@id='button_managedField']]").findWhenNeeded(this);
+//        WebElement additionalKeyDataFieldParentSpan = Locator.xpath("//span[descendant::input[@id='button_dataField']]").findWhenNeeded(this);
+//        WebElement additionalKeyNoneParentSpan = Locator.xpath("//span[descendant::input[@id='button_none']]").findWhenNeeded(this);
+//
+//        Locator.XPathLocator additionalKeyDisabledNone = Locator.xpath("//span[contains(@class,'gwt-RadioButton-disabled')]/input[@id='button_none']");
+//        Locator.XPathLocator additionalKeyDataFieldDisabled = Locator.xpath("//span[contains(@class,'gwt-RadioButton-disabled')]/input[@id='button_none']");
+//        Locator.XPathLocator additionalKeyMangedFieldDisabled = Locator.xpath("//span[contains(@class,'gwt-RadioButton-disabled')]/input[@id='button_none']");
+//
+//        PropertiesEditor fieldsEditor = PropertiesEditor.PropertiesEditor(getDriver()).withTitleContaining("Dataset Fields").findWhenNeeded();
+//    }
+//}
```

Comment:
Looks like this whole file has been commented out, should it be deleted?

Reply:
Yes.  I left it in at first so I could refer to the old one while doing conversion.